locomotion-sdk/src/pages/AuthScreens/StartScreen/index.js

```diff
@@ -56,16 +64,28 @@ const StartScreen = ({ navigation }) => {
       <PageContainer>
         {!webViewWindow ? (
           <>
-            <LogoContainer>
-              <Logo source={logo} />
-            </LogoContainer>
+            <InfoContainer>
+              <LogoContainer>
+                <Logo source={operation.logo} />
+              </LogoContainer>
+              <OperationName>{operation.name}</OperationName>
+              {operation.subName && <OperationSubName>{operation.subName}</OperationSubName>}
+            </InfoContainer>
             <ButtonsContainer>
               <StartButton
+                dark
                 onPress={() => {
                   navigation.navigate('Phone');
                 }}
               >
-                <ButtonText>{i18n.t('login.getStarted')}</ButtonText>
+                <ButtonText dark>{i18n.t('login.signUp')}</ButtonText>
+              </StartButton>
+              <StartButton
+                onPress={() => {
+                  navigation.navigate('Phone');
```

Comment:
use the navigation service

Reply:
you mean create a navigation service?

Reply:
i made a new branch and created a navigation service. ill switch over the all the uses there